src/js/components/Form/Field.jsx

```diff
@@ -168,6 +171,17 @@ function Field({
             value={value}
           />
         )}
+        {type === 'project' && (
+          <ProjectPicker
+            disabled={disabled}
+            name={name}
+            readOnly={readOnly}
+            required={required}
+            value={value}
+            onChange={onChange}
+            onError={onError}
```

Comment:
I'm on the fence about handling errors differently for this specific field type.  IIRC, the other field components expect that the "on change" method detects the error and passes it back as the `errorMessage` prop.

Reply:
The `errorMessage` prop is for displaying validation errors - they make the field border red and display some text below. This is for request errors propagating from the backend, that `ProjectPicker` may get when it fetches projects, so it's different. When this happens it needs to notify its parent so that it can display a different a more bold and in-your-face error that something went horribly wrong.

This did make me realize tho that I'm fetching the namespaces and project types unnecessarily in`ProjectPicker`, as they're already available in the settings metadata.